### PR TITLE
chore(redirectd): redirectd is bazelified

### LINKED ISSUE #11302: bazel: Bazelify Python redirectd

### PR DESCRIPTION
**Pull request by LKreutzer**
Signed-off-by: Lars Kreutzer <lars.kreutzer@tngtech.com>

## Summary

- Module redirectd is bazelified including the test. 
- Minor refactoring of mobilityd dependencies -> move dependencies to where they are actually needed.

## Test Plan

-  Run redirectd: 
   - `bazel run //lte/gateway/python/magma/redirectd:redirectd`
   - Note 1: This is expected to fail in the dev container, because needed services are not running.
   - Note 2: Runs in the magma-VM
- Run test: 
  - `bazel test //lte/gateway/python/magma/redirectd/tests:all`

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/mobilityd/BUILD.bazel [hunk 1]
```diff
@@ -20,29 +20,12 @@ DEPS = [
     "//lte/protos:keyval_python_proto",
     "//lte/protos:mconfigs_python_proto",
     "//lte/protos:mobilityd_python_grpc",
-    "//lte/protos:mobilityd_python_proto",
     "//lte/protos:subscriberdb_python_grpc",
-    "//lte/protos:subscriberdb_python_proto",
     "//lte/gateway/python/magma/subscriberdb:sid",
-    "//orc8r/gateway/python/magma/common:metrics",
+    "//orc8r/gateway/python/magma/common:rpc_utils",
```
Comment (themarwhal) [inline lte/gateway/python/magma/mobilityd/BUILD.bazel:25]: What's happening here? were the deleted deps actually dependencies for rpc_utils?

Comment (LKreutzer) [reply lte/gateway/python/magma/mobilityd/BUILD.bazel:25]: This depends on the deleted dependency. In the case of "common:metrics" it was actually a dependency of "common:service" and thus it is now included in "common:service" as a source file. In other cases the dependency was moved to where it originated.

What we did: 
- Remove all dependencies from mobilityd
- Run `bazel run //...:mobilityd`
- Add missing dependencies at origin of error

Comment (themarwhal) [reply lte/gateway/python/magma/mobilityd/BUILD.bazel:25]: ah great! That's better. Thanks for the explanation.

**Review by themarwhal — APPROVED** [review]
lgtm!

File: lte/gateway/python/magma/redirectd/tests/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# limitations under the License.
+
+load("@python_deps//:requirements.bzl", "requirement")
+load("//bazel:python_test.bzl", "pytest_test")
+
+COMMON_TEST_DEPS = [
+    "//lte/gateway/python/magma/redirectd:redirectd",
+    requirement("pytest"),
+    requirement("pytest-cov"),
+    requirement("coverage-lcov"),
```
Comment (themarwhal) [inline lte/gateway/python/magma/redirectd/tests/BUILD.bazel]: maybe we can pull these out to `//bazel:python_test.bzl` if they're always needed for tests.

File: lte/gateway/python/magma/redirectd/tests/BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+]
+
+COMMON_TAGS = [
+    "lte",
+    "redirectd",
+    "unittest",
+]
```
Comment (themarwhal) [inline lte/gateway/python/magma/redirectd/tests/BUILD.bazel]: These don't seem necessary

Comment (LKreutzer) [reply lte/gateway/python/magma/redirectd/tests/BUILD.bazel]: Oh no! We forgot to change this here - will be changed!

File: lte/gateway/python/magma/redirectd/tests/BUILD.bazel (hunk 2 above)
Comment (LKreutzer) [reply lte/gateway/python/magma/redirectd/tests/BUILD.bazel]: Good idea, pushed!

File: lte/gateway/python/magma/redirectd/tests/BUILD.bazel (hunk 3 above)
Comment (LKreutzer) [reply lte/gateway/python/magma/redirectd/tests/BUILD.bazel]: Done.

**Review by nstng — APPROVED** [review]
done in pairing

**Review by themarwhal — APPROVED** [review]

**Review by ajahl — APPROVED** [review]
lgtm

**Review by ardzoht — APPROVED** [review]
`lte` changes lgtm

**Pull request merged**